commands/outliers.js

```diff
@@ -41,7 +45,8 @@ const cmd = {
   args: [{name: 'database', optional: true}],
   flags: [
     {name: 'reset', description: 'resets statistics gathered by pg_stat_statements'},
-    {name: 'truncate', char: 't', description: 'truncate queries to 40 characters'}
+    {name: 'truncate', char: 't', description: 'truncate queries to 40 characters'},
+    {name: 'num', char: 'n', description: 'the number of queries to display (default: 10)'}
```

Comment:
You have `num` here but `flags.limit` above--I think either one works unless we're using one or the other already in `heroku-pg` or `heroku-pg-extras` somewhere. I can check that.

Reply:
We're not using either; I think either is fine.

Reply:
Good catch. I totally missed updating the `context.flags.limit` when I renamed the flag from `limit` to `num`. I was trying to be a bit more descriptive in the naming of the flag `-n` to control the number of results seems like a relatively common approach, rather the limit (which was implementation driven naming).